airflow-core/docs/authoring-and-scheduling/serializers.rst

```diff
@@ -87,7 +87,7 @@ Registered
 
     from typing import TYPE_CHECKING
 
-    from airflow.utils.module_loading import qualname
+    from airflow._shared.module_loading import qualname
```

Comment:
Let's not have shared paths in docs when possible. For this case, since we are going to move serde and serializers to task sdk, we can have it consume the `airflow.sdk` path instead by re-exporting it better